js/directives/accordion.directive.js

```diff
@@ -0,0 +1,22 @@
+app.directive('custAccordion', function ($timeout) {
+    return {
+        templateUrl: 'template/accordionTemplate.html',
+        restrict: 'E',
+        scope: true,
+        link: function (scope, elem, attrs) {
+            var redrawGrid = function () {
+                $timeout(function () {
+                    $(window).resize();
```

Comment:
Why 2  $(window).resize(); ???
And I think instead of adding to timeout u can add it to the post link phase